internal/checker/cloudformation/rules/logging/verify_s3_bucket_logging_enabled.rego

```diff
@@ -0,0 +1,50 @@
+# METADATA
+# title: "Verify S3 bucket access logging is enabled"
+# description: "This policy ensures thorough auditing for all files and directories stored within an S3 bucket."
+# scope: package
+# related_resources:
+# - https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-s3-bucket.html
+# custom:
+#   id: CB_CFT_009
+#   severity: LOW
+package lib.cloudformation.CB_CFT_009
+import future.keywords.in
+
+resource := "AWS::S3::Bucket"
+
+is_valid {
+	some resources in input.Resources
+	resources.Type == resource
+}
+
```

Comment:
update logic to 
pass[properties] {
    is_valid
    some resources in input.Resources
    some properties in resources
    "LoggingConfiguration" in object.keys(properties)
}